src/elements/content-sidebar/additional-tabs/AdditionalTab.js

```diff
@@ -18,20 +18,51 @@ type Props = {
     onImageLoad: () => void,
 } & AdditionalSidebarTab;
 
-const AdditionalTab = ({ callback: callbackFn = noop, iconUrl, id, isLoading, onImageLoad, title, ...rest }: Props) => {
-    const className = classNames('bdl-AdditionalTab', { 'bdl-AdditionalTab--hidden': isLoading });
-
-    return (
-        <Tooltip position="middle-left" text={title}>
-            <PlainButton className={className} type="button" onClick={() => callbackFn({ id, callbackData: rest })}>
-                {id && id > 0 && iconUrl ? (
-                    <img className="bdl-AdditionalTab-icon" src={iconUrl} onLoad={onImageLoad} alt={title} />
-                ) : (
-                    <IconEllipsis className="bdl-AdditionalTab-moreIcon" />
-                )}
-            </PlainButton>
-        </Tooltip>
-    );
+type State = {
+    isErrored: boolean,
 };
 
+class AdditionalTab extends React.PureComponent<Props, State> {
+    state = {
+        isErrored: false,
+    };
+
+    onImageError = () => {
+        this.props.onImageLoad();
```

Comment:
Is the `onImageLoad` callback expecting any arguments? Is there anything passed from the onload event?

Reply:
onImageLoad adds to the global count of loaded images, so it doesn't need any specifics from an individual load. Based on the MDN example I was looking at they don't expect any arguments to come back from that event.